exo-deploy/src/cli.ls

```diff
@@ -2,14 +2,31 @@ require! {
   'chalk' : {cyan, green}
   './docker/docker' : Docker
   'exosphere-shared' : {Logger}
+  'inquirer'
   'path'
   'require-yaml'
 }
 
+command-flag = process.argv[2]
 app-config = require path.join(process.cwd!, 'application.yml')
-console.log "Deploying #{green app-config.name} #{cyan app-config.version}\n"
 logger = new Logger
-new Docker app-config, logger
-  ..dockerhub-push (err) ~>
-    | err => return logger.log name: 'exo-deploy', text: err.message
-    ..start!
+docker = new Docker app-config, logger
+if command-flag is '--nuke'
+  console.log "You are about to nuke #{green app-config.name} #{cyan app-config.version}\n"
+
+  question =
+    type: 'list'
+    name: 'continue'
+    message: 'Are you sure?'
+    choices: ['yes', 'no']
+  inquirer.prompt([question]).then (answer) ->
+    if answer.continue == 'no'
+      console.log '\nAborting ...\n'
+      process.exit!
```

Comment:
Exit with code 2, since this isn't a normal exit. Exit codes are:
- 0 it worked
- 1 an error occurred
- 2 user aborted